### PR TITLE
fix: invalid grammar for `var` in classes

### PR DESCRIPTION
**Pull request by eladb**
Fix the tree sitter grammer to match the parsing tree of `var` in `class` to be the same as `let`.

This is currently broken in our build.

Also - update the workflow files to *exclude* projects from build and test.

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Review by Chriscbr — APPROVED** [review]

File: .github/workflows/build.yml
Comment (eladb) [inline .github/workflows/build.yml:129]: It seems like this list of projects effectively builds all the projects. no?

Comment (MarkMcCulloh) [reply .github/workflows/build.yml:129]: It's specifically leaving out the native-based projects (vscode and lsp)

Comment (MarkMcCulloh) [reply .github/workflows/build.yml:129]: So once this is merged, it'll take a lot longer

File: .github/workflows/build.yml
Comment (MarkMcCulloh) [inline .github/workflows/build.yml:137]: this will cause wingc to do a full native build. I would suggest adding the specific new projects you want here instead.

File: .github/workflows/build.yml
Comment (eladb) [reply .github/workflows/build.yml:129]: I'll exclude them.

Comment (MarkMcCulloh) [reply .github/workflows/build.yml:129]: For the purpose of this PR, I don't think this line needs to be changed

**Review by MarkMcCulloh — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml
Comment (eladb) [reply .github/workflows/build.yml:129]: added "exclude"

**Comment by MarkMcCulloh** [comment]
side note: It would probably be useful to do workflow changes in a fork and preview it there first, otherwise you’ll have to wait for it to merge.

File: .github/workflows/build.yml
Comment (MarkMcCulloh) [reply .github/workflows/build.yml:138]: the test target for wingc runs cargo test, which uses the native platform target by default

Comment (eladb) [reply .github/workflows/build.yml:138]: As discussed, we are already running `wingc` tests.

**Review by MarkMcCulloh — APPROVED** [review]

**Pull request merged**
